lib/dotcom_web/controllers/helpers.ex

```diff
@@ -122,11 +122,11 @@ defmodule DotcomWeb.ControllerHelpers do
   """
   @spec forward_static_file(Conn.t(), String.t()) :: Conn.t()
   def forward_static_file(conn, url) do
-    case client() |> @req.get(url: url) do
-      {:ok, %{status: 200, body: body, headers: headers}} ->
+    case client() |> @req.get(url: url, cache: true) do
```

Comment:
What's the cache timing on this?

Reply:
N/A, it uses the `if-modified-since` header! ([doc](https://hexdocs.pm/req/0.5.1/Req.Steps.html#cache/1))